### ISSUE TITLE
widgets not automatically installed?

### ISSUE THREAD

**Issue by LocutusOfBorg** [pyqtgraph/pyqtgraph#1100]
Some files, e.g. widgets are not automatically installed in some configurations, not sure why and how to debug it

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=939571

right now we hardcoded the file list instead of relying on the helper, we don't know if this is a python bug or something related to pyqtgraph...

Can you please help?

**Comment by LocutusOfBorg** [comment]
this might work on the latest 0.11.0-rc0, so feel free to close it if it doesn't ring any bell

**Comment by ksunden** [comment]
This was a bug with the Debian packages, specifically (somehow was missing the last item in an alphabetic list), It was fixed with debian package 0.10.0-5 (Debian unstable has since been updated to the release candidate).

**Comment by j9ac9k** [comment]
@ksunden are the debian packages a popular method of distributing pyqtgraph?  Most people I know interact with it via the wheel hosted on PyPI.  

The reason I'm asking is that in the version following 0.11; we will be implementing NEP-29, which will limit the python and numpy backward compatibility support.  Along the same lines, we will be limiting the versions we support/test for Qt bindings.  I have been thinking to only support Qt 5.12+, but if Debian unstable  uses an older version of Qt, this would be good to know.

**Comment by ksunden** [comment]
@j9ac9k Note this is the debian sid/unstable (looks like testing/bullseye as well) that I'm referring to, where 5.12 is the current libqt5.

I'm not sure how common it is, I'd guess less common for devs, but if there are apps which rely on pyqtgraph (gnuradio is one) then the debian package `python3-pyqtgraph` is a dependency, which will be pulled in by apt installing.

In my opinion, devs should typically be using venvs and pip (or conda), not distro provided libs, but I'm sure there are some who just use what's packaged in the debian repos.

@georgesk I believe is the member of the Debian Science Team who has packaged the most recent version, he may know more

**Comment by j9ac9k** [comment]
if the current `libqt5` is 5.12, we're all good 👍 

@georgesk As we're changing up the backward compatibility model of pyqtgraph, I do not want to break the package for existing users (or at least try to minimize the hurt), users of `python3-pyqtgraph` are largely unknown to us; so we'd like to hear from you regarding what your needs are.  If you're good with `pyqtgraph` supporting python 3.6+, Qt 5.12+ and adopting NEP-29 (numpy 1.15+ starting January 7th of this year) then everything will likely keep working...

**Comment by LocutusOfBorg** [comment]
I think on Debian side we are good for numpy, qt and python.
@ksunden I'm the person who uploaded that version in Debian, but that version was an hacky workaround, this is why I reported it upstream
```
   [ Georges Khaznadar ]
   * For some mysterious reason, the expressions used to build the list of
     python files to package misses the last directory in the list; when I
     call *the same* expressions in a Python shell, it does not fail. So I
     made a quick & &dirty fix: I hardcoded the list of files to package,
     which closes: #939571
```
not sure why it works now.
